src/ui/Button/index.tsx

```diff
@@ -18,7 +18,7 @@ export const Button = ({
     style={[
       styles.button,
       styles[variant],
-      disabled ? styles.disabled : {},
+      disabled && styles[`${variant}Disabled`],
```

Comment:
Maybe to check with @Ldoppea who also worked on this here https://github.com/cozy/cozy-flagship-app/pull/1062/commits/272093d81998c6fafd3c622a73862fb2cfa0b087

Reply:
bit hard to update, @Ldoppea I think the best way to handle that is you just overwrite my styles during conflict. I update to follow same naming convention in styles though